### DISCUSSION TITLE
Desktop notifications duration

### DISCUSSION THREAD

**Discussion by dpetka2001** [kovidgoyal/kitty#6976] [Q&A]
I was reading about [desktop notifications](https://sw.kovidgoyal.net/kitty/desktop-notifications/) and I was trying the code in the examples
```bash
printf '\x1b]99;i=1:d=0;Hello world\x1b\\'
printf '\x1b]99;i=1:d=1:p=body;This is cool\x1b\\'
```
and I'm wondering if there's some kind of additional flag that controls the duration of the notification's visibility. I read through the link in the docs, but couldn't find anything.

**Comment by kovidgoyal** [comment]
There are no parameters other than documented ones.

**Reply by dpetka2001** [reply]
Is there some other way to extend the duration of the notifications visibility to your knowledge ?
